src/textProcessing/ProcessText.py

```diff
@@ -46,7 +46,11 @@ def getUserName(userInput):
 
     @staticmethod
     def isAffirmative(userInput):
-        return False
+        if userInput == 'yes' or 'ya' or 'that is right' or 'correct' or 'right':
```

Comment:
affirmative words should be in array, and user input should be affirmative iff it contains an affirmative word or phrase